staker/block_validator.go

```diff
@@ -172,7 +172,12 @@ func (c *BlockValidatorConfig) Validate() error {
 }
 
 type BlockValidatorDangerousConfig struct {
-	ResetBlockValidation bool `koanf:"reset-block-validation"`
+	ResetBlockValidation bool               `koanf:"reset-block-validation"`
+	Revalidation         RevalidationConfig `koanf:"re-validation"`
+}
+
+type RevalidationConfig struct {
+	StartBlock uint64 `koanf:"start-block"`
```

Comment:
For completeness, how about an optional `end-block` as well?  That would make reproduceability easier.  If `end-block` is specified, should probably require user to supply `then-quit` as well?

Reply:
This is exactly what we wanted to do in https://linear.app/offchain-labs/issue/NIT-3019/support-re-validation-of-blocks-advanced

Since the changes are small adding it to this PR only instead of opening a new PR.